config.go

```diff
@@ -88,7 +89,9 @@ func (c *Config) LoadEnv() error {
 	}
 
 	if v := os.Getenv("NGX_OMNIAUTH_SESSION_SECRET"); v != "" {
-		c.Secrets = []*string{&v}
+		for _, v := range strings.Split(v, ",") {
+			c.Secrets = append(c.Secrets, &v)
```

Comment:
The scope of `v` is too wide.
This code should be like the following.

```go
v := v
c.Secrets = append(c.Secrets, &v)
```

Reply:
Thanks. I fixed it.